app.arc

```diff
@@ -81,6 +81,10 @@ circulars_subscriptions
   sub *String
   name circularsSubscriptionsBySub
 
+legacy_users
+  receive *Number
```

Comment:
DynamoDB does have a boolean type. Does it support boolean fields as partition keys?